requirements-dev.txt

```diff
@@ -15,6 +15,13 @@ mccabe==0.6.1
 pep8==1.7.0
 pyflakes==1.5.0
 
+pytest==3.1.1
+pytest-xdist==1.18.1
+pytest-cov==2.5.1
+pytest-mongodb==2.1.1
```

Comment:
I think items like `pytest-mongodb` need to be put into the `setup.py` of `pytest_girder`.